front/js/script.js

```diff
@@ -0,0 +1,30 @@
+//The function below displays all products img, h3 and p on the home page
+
+fetch("http://localhost:3000/api/products")
+    .then(function(res) {
+        if (res.ok) {
+            return res.json();
+        }
+    })
+    .then(function(value) {
+            for (let i in value) {
+                const displayedProduct = document.getElementById("items");
+                const displayedProductArticle = document.createElement("article");
```

Comment:
Are you not missing the link which should encompass the whole article ? To go the product page when clicking on article?

Reply:
Added anchor encompassing. Link is obtained by concatenating the default product link with values from the array.